tests/page-objects/user-settings/user-settings.wdio.page.js

```diff
@@ -0,0 +1,22 @@
+const submiButton = () => $('.btn.submit.btn-primary');
+
+const openEditSettings = async () => {
+  const links = await $('.content .configuration');
+  await links.waitForDisplayed();
+  await links.$$('.btn-link')[1].click();
```

Comment:
This fails for me locally with: 
```
TypeError: Cannot read property 'click' of undefined
    at Object.openEditSettings (/home/diana/projects/medic-second/tests/page-objects/user-settings/user-settings.wdio.page.js:6:33)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Context.<anonymous> (/home/diana/projects/medic-second/tests/e2e/translations/new-language.wdio-spec.js:53:5)
    at async Context.executeAsync (/home/diana/projects/medic-second/node_modules/@wdio/utils/build/shim.js:136:16)
    at async Context.testFrameworkFnWrapper (/home/diana/projects/medic-second/node_modules/@wdio/utils/build/test-framework/testFnWrapper.js:52:18)

```

I think this fixes it: 
```suggestion
  await (await links.$$('.btn-link'))[1].click();
```